x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_empty.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { useValues } from 'kea';
+
+import { i18n } from '@kbn/i18n';
+import {
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiPageContent,
+  EuiPageContentHeader,
+  EuiPageContentBody,
+  EuiTitle,
+  EuiText,
+  EuiButton,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { EngineLogic } from '../engine';
+
+import { DOCS_PREFIX } from '../../routes';
+import {
+  DOCUMENT_CREATION_DESCRIPTION,
+  DOCUMENT_API_INDEXING_TITLE,
+  DOCUMENT_API_INDEXING_DESCRIPTION,
+} from '../document_creation/constants';
+// TODO
+// import { DocumentCreationButtons, CodeExample } from '../document_creation'
+
+export const EmptyEngineOverview: React.FC = () => {
+  const { engineName } = useValues(EngineLogic);
+
+  return (
+    <>
+      <EuiPageHeader>
+        <EuiPageHeaderSection>
+          <EuiTitle size="l">
+            <h1>
+              {i18n.translate('xpack.enterpriseSearch.appSearch.engine.overview.empty.heading', {
+                defaultMessage: 'Engine Setup',
+              })}
+            </h1>
+          </EuiTitle>
+        </EuiPageHeaderSection>
+        <EuiPageHeaderSection>
+          <EuiButton href={`${DOCS_PREFIX}/index.html`} target="_blank">
+            {i18n.translate(
+              'xpack.enterpriseSearch.appSearch.engine.overview.empty.headingAction',
+              { defaultMessage: 'View Documentation' }
+            )}
+          </EuiButton>
+        </EuiPageHeaderSection>
+      </EuiPageHeader>
+      <EuiPageContent>
+        <EuiPageContentHeader>
+          <EuiTitle>
+            <h2>
+              {i18n.translate('xpack.enterpriseSearch.appSearch.engine.overview.empty.subheading', {
+                defaultMessage: 'Set up the “{engineName}” Engine',
+                values: { engineName },
+              })}
+            </h2>
+          </EuiTitle>
+        </EuiPageContentHeader>
+        <EuiPageContentBody>
+          <EuiText color="subdued">
+            <p>{DOCUMENT_CREATION_DESCRIPTION}</p>
```

Comment:
I just discovered this (and the API indexing title/description) is reused word for word in the Index Documents modal, so I pulled it out to a constants file:

<img width="738" alt="" src="https://user-images.githubusercontent.com/549407/99342966-d9621080-2841-11eb-9c36-dc0e9070d0d3.png">

I'll confess I don't super love the end effect and it makes the page a tad hard to "read" without looking at the actual compiled app, but what can you do 🤷